### PR TITLE
Create pr.yaml

### PR DESCRIPTION
**Pull request by hamelsmu**
Triggering text:

`This pull request represents a submission to the codesearchnet benchmark.`

### CODE REVIEW

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        API_VERSION=v3
+        API_HEADER="Accept: application/vnd.github.${API_VERSION}+json"
+        AUTH_HEADER="Authorization: token ${GITHUB_TOKEN}"
+        MESSAGE="@${USER} Thank you for your benchmark submission!  Can you please tell us a little bit about your approach and how you trained your models etc?  What modifications, if any, did you make on the baseline models provided in this repository? Thanks!"
+        # Create a comment with APIv3 # POST /repos/:owner/:repo/issues/:issue_number/comments
+        curl -XPOST -sSL \
+        -d "{\"body\": \"$MESSAGE\"}" \
+        -H "${AUTH_HEADER}" \
+        -H "${API_HEADER}" \
+        "${URI}/repos/${GITHUB_REPOSITORY}/issues/${ISSUE_NUMBER}/comments"
```
Comment (mallamanis) [inline .github/workflows/pr.yaml]: @hamelsmu you'll know the best, but is `issues/${ISSUE_NUMBER}` right? Should it be something with PRs?
https://developer.github.com/v3/pulls/comments/#create-a-comment
```suggestion
        "${URI}/repos/${GITHUB_REPOSITORY}/pulls/${PR_NUMBER}/comments"
```
(don't know the actual name that `PR_NUMBER` is exposed under...)

Comment (hamelsmu) [reply .github/workflows/pr.yaml]: Good catch

Comment (hamelsmu) [reply .github/workflows/pr.yaml]: good catch ... fixing now

Comment (hamelsmu) [reply .github/workflows/pr.yaml]: Nevermind.  that endpoint is for PR Review Comments, not the pull request itself.  In GitHub, confusingly, a PR == Issue.  

https://developer.github.com/v3/pulls/:

```
The Pull Request API allows you to list, view, edit, create, and even merge pull requests. Comments on pull requests can be managed via the Issue Comments API.
```

Comment (hamelsmu) [reply .github/workflows/pr.yaml]: Same thing goes for assignees 😆 

image

**Comment by hamelsmu** [comment]
I'm closing this PR temporarily an re-opening it so I can trigger this action and see if it fires.  Sorry for the noise.

**Comment by hamelsmu** [comment]
ok trying one more time ... found an error last time.

**Comment by hamelsmu** [comment]
I guess what I want is reviewer, not assignnee 🤦‍♂ ok trying again

**Comment by hamelsmu** [comment]
doing this hopefully the last time for the triggers

**Comment by hamelsmu** [comment]
Ok merging now, confident that it works, finally.

Thank you everyone for your patience in this PR and sorry for the notification spam it may have caused you.  My apologies.

image

**Pull request merged**